tests/network/general/test_ip_family_services.py

```
@@ -93,6 +85,35 @@ def virtctl_expose_service(
+):
+    ip_family_policy = request.param
+    svc = _create_virtctl_expose_service(
+        ip_family_policy=ip_family_policy,
+        running_vm_for_exposure=running_vm_for_exposure,
+        unprivileged_client=unprivileged_client,
+    )
+    yield svc
+    svc.clean_up()
+
+
+@pytest.fixture()
```

Ruff (0.14.10)

Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)